plugins/inputs/exec/exec.go

```diff
@@ -49,19 +50,24 @@ type Exec struct {
 
 func NewExec() *Exec {
 	return &Exec{
-		runner:  CommandRunner{},
+		runner:  &CommandRunner{},
```

Comment:
Since this needs to change to a pointer, can you update other methods to use a pointer? Ex: the truncate() method and any others

Reply:
all done!